components/StarkNet/modules/getting_started/pages/environment_setup.adoc

```diff
@@ -1,160 +1,208 @@
 [id="environment"]
 
-[#setup]
-= Setting up your local development environment
+# Setting up the environment
 
-This section will cover three steps: 
+## Overview
 
-* Installing prerequisites
-* Installing the `cairo-lang` CLI
-* Installing the `cairo` Cairo 1 compiler
+Setting up your local development environment to develop on Starknet involves the following steps:
 
-== Prerequisites
+* xref:#_install_the_prerequisites[Install the prerequisites]
+* xref:#_install_the_additional_dependencies[Install the additional dependencies]
+* xref:#_install_the_cairo_cli[Install the Cairo CLI]
+* xref:#_install_the_cairo_compiler[Install the Cairo compiler]
 
-In order to install and use `cairo` and `cairo-lang`, install the following: +
 
-* https://www.python.org/downloads/release/python-390/[Python 3.9] +
-* https://www.rust-lang.org/tools/install[Rust] +
+## Install the prerequisites
 
-== cairo-lang CLI Installation
+We recommend using a link:https://docs.python.org/3/library/venv.html[Python virtual environment]
+for your development work.
 
-=== Step 1: Set up your virtual environment
+To create and enter the Python virtual environment, you have two options:
 
-We recommend working inside a Python virtual environment, but you can also install the `cairo`
-package directly. To create and enter the virtual environment, type:
+* xref:#_option_1__activate_your_cairo_virtual_environment_manually[Activate your Cairo virtual environment manually]
+* xref:#_option_2__use_the_python_environment_manager[Use the Python environment manager]
+
+
+### Option 1 - Activate your Cairo virtual environment manually
+
+Run the following two commands to manually create and activate a Cairo virtual environment:
 
 [,Bash]
 ----
-#Activate your cairo virtual environment
 python3.9 -m venv ~/cairo_venv
 source ~/cairo_venv/bin/activate
 ----
 
-Alternatively, you can also use the `pyenv` environment manager to create a virtual environment. To do so, run the following commands:
+After creating and activating the virtual environment, you should see `(cairo_venv)` in the command line prompt, indicating that the virtual environment is active, for example:
+
+[,Bash]
+----
+(cairo_venv)
+----
+
+
+### Option 2 - Use the Python environment manager
+
+Alternatively, you can use the Python environment manager to create a virtual environment. Simply run the following commands:
```

Comment:
```suggestion
Alternatively, you can use the Python environment manager to create a virtual environment:
```

Avoid saying things like "Simply do this" or "Just do that" because if something is not simple for a particular user, it can make them feel stupid, and for users for whom it _is_ simple, adding the word doesn't have any benefit.